README.md

```diff
@@ -6,5 +6,18 @@ Fork and clone this repository.
 
 ### Questions
 1. Explain why Rails exists and what it's used for.
+
+Rails exists as a framework to combine Ruby with HTML, CSS, and JavaScript to create a web application.
+
+
 1. Using a marker, pen, or pencil, sketch the components of an (R)MVC architecture, how they interconnect, and which components have what responsibilities. Take a photo of your drawing and add it to the `submissions` directory inside your clone.
+
```

Comment:
So, the overflow isn't far off, but it's wrong on a couple of the specifics. The model doesn't contact the view. In fact, it doesn't actually contact anything but the database. The controller is the top-level code, calling model methods , using the results to build the view, and then handing off the finished product to the server.